### ISSUE TITLE
Don't hide the help again after a few seconds

### ISSUE THREAD

**Issue by porky11** [Lecrapouille/TimedPetriNetEditor#5]
It's annoying to press "h" everytime I want to see the help. As a beginner, it seems helpful to be able to see the help all the time.
So "h" should just toggle it instead. Even if I just want to look at it for a few seconds, I only have to press "h" twice, even if I want to look at it for longer.

**Comment by Lecrapouille** [comment]
@porky11 yes, I'm aware of this problem. The GUI is quite limited, and some results are also temporarily displayed. So this is a more general drawback. Are you using Linux ? The help is displayed on the console when the application is started.
